lib/Conversation/Comment.js

```diff
@@ -88,62 +75,62 @@ class Comment extends Component {
+        <Person person={author} className="conversation" />
 
-                {userCanEdit && (
-                  <span className="conversation__actions">
-                    <div
-                      className="button button--link button--link-default button--collapse conversation__meta conversation__edit-link"
-                      onClick={this.showEditForm}
-                      onKeyUp={this.handleEditKeyPress}
-                      role="button"
-                      tabIndex={0}
-                    >
-                      Edit
-                    </div>
-                    <Button
-                      className="conversation__meta"
-                      types={['link', 'link-danger', 'collapse']}
-                      clickHandler={() => this.removeComment(false)}
-                    >
-                      Delete
-                    </Button>
-                  </span>
-                )}
-              </div>
-            </div>
-          </div>
-        )}
-
-        {userCanEdit &&
-          this.state.showEditForm && (
-            <CommentForm
-              id={this.props.id}
-              onSubmit={this.editComment}
-              onCancel={this.hideEditForm}
-              author={author}
-              value={body}
-              placeholder=""
-              focusOnMount
-              isSubmitting={false}
-              onCommentChange={this.props.onCommentChange}
-              onRowCountChange={this.props.onRowCountChange}
-              editing
-            />
+        <div className="conversation__comment-body">
+          {!this.state.showEditForm && (
+            <p className="conversation__text">
+              <Linkify>{body}</Linkify>
+            </p>
           )}
 
+          {userCanEdit &&
+            this.state.showEditForm && (
+              <CommentForm
+                id={this.props.id}
+                onSubmit={this.editComment}
+                onCancel={this.hideEditForm}
+                value={body}
+                author={author}
+                placeholder=""
+                focusOnMount
+                isSubmitting={false}
+                onCommentChange={this.props.onCommentChange}
+                onRowCountChange={this.props.onRowCountChange}
+                showAuthor={false}
+                editing
+              />
+            )}
+
+          <div className="conversation__meta-wrapper">
+            {!this.state.showEditForm && (
+              <span className="conversation__meta conversation__date-text">
+                {createdAt}
+              </span>
+            )}
+
+            {userCanEdit &&
+              !this.state.showEditForm && (
+                <span className="conversation__actions">
+                  <Button
+                    className="conversation__meta conversation__edit-link"
+                    types={['link', 'link-default', 'collapse']}
+                    clickHandler={this.showEditForm}
+                  >
```

Comment:
I made the edit Button a div previously due to how Quill's selection change doesn't get triggered when changing focus to a button element. Have you found a different work around? It wasn't a super obvious bug, but it would cause some weirdness

Reply:
Ah ok, I forgot about that. Maybe we should add a test to document why this is the case. Will add a test now to say that we need this to be the case due to 3rd part libraries.

Reply:
Sounds good! It is a bit of an annoying one 😅